src/appleseed.studio/mainwindow/mainwindow.cpp

```diff
@@ -870,19 +909,14 @@ namespace
     }
 }
 
-bool MainWindow::can_close_project()
+bool MainWindow::attempt_close_project()
```

Comment:
I will revert it back to old can_close_project function as I don't use separated part anyway